src/java/org/apache/cassandra/config/CassandraDuration.java

```diff
@@ -27,22 +27,23 @@
 import com.google.common.primitives.Ints;
 
 /**
- * Represents a positive time duration.
+ * Represents a positive time duration. Wrapper class for Cassandra duration configuration parameters, providing to the
+ * users the opportunity to be able to provide config with a unit of their choice in cassandra.yaml as per the available
+ * options. (CASSANDRA-15234)
  */
-public final class Duration
+public final class CassandraDuration
```

Comment:
If we're still accepting naming suggestions, I'll throw `DurationSpec` and `DurationWrapper` into the mix (mostly to avoid the `CassandraX` pattern if possible).

Reply:
I am accepting. I am all in for `DurationSpec`. I will leave it as a note for now until we confirm the change with the other reviewers so I don't have to refactor a few times. 
Also, we need then `BitRatesSpec` and `DataStorageSpec`